package.json

```diff
@@ -49,11 +49,12 @@
   "scripts": {
     "docs:build": "gitbook install && gitbook build",
     "docs:watch": "gitbook install && gitbook serve",
-    "prepublish": "nwb build",
+    "prepublish": "nwb build && npm run types",
     "release": "semantic-release pre && npm publish && semantic-release post",
     "test": "node umd/skatejs.js && semistandard && npm run test:ts && nwb test -- --ci",
     "test:ts": "tsfmt -r && tsc -p ./",
-    "test:watch": "nwb test --server"
+    "test:watch": "nwb test --server",
+    "types": "../../treshugart/get-typed/bin/get-typed.js src/index.js --dest umd/skatejs"
```

Comment:
I would like to see generated TS definitions. I highly doubt that the "new tool" will correctly generate proper mixin definition for class type extension. If they do on the other hand then kudos, high five, mic drop.

I'm back from vacation, I'll look into this and other things this week.

maybe off topic but I hope flow works OK on windows, will test that as well. ( build is not working on win at the moment )